pyproject.toml

```diff
@@ -77,9 +77,10 @@ dependencies = [
     "utm",
     "hopp>=3.2.0",
     "orbit-nrel==1.1",
-    "openmdao[all]",
+    "openmdao",
     "ProFAST",
     "CoolProp",
+    "ruamel.yaml",
```

Comment:
I don't see this being used anywhere.

Reply:
I thought I thought the same thing Rob and removed this a few months back and experienced CI issues when installing in GitHub Actions. Would be worth testing on a branch.

Reply:
Definitely worth digging into a bit more. I'm not really opposed to it since the library adds some nice features to YAML if we were to use them, but it felt like adding a random library without your explanation.

Reply:
Sorry for not giving more context here. We use `ruamel` in the validators for the yamls in the new H2I framework. This _should've_ been added as a package when I added the new framework examples, but I did not catch it until this PR when I make CI run the new examples. Hence, adding this package.

Reply:
Good to know!